spec/paperclip/geometry_spec.rb

```diff
@@ -82,7 +82,7 @@
       assert_equal 456, @upper.height
     end
 
-    ['>', '<', '#', '@', '%', '^', '!', nil].each do |mod|
+    ['>', '<', '#', '@', '@>', '>@', '%', '^', '!', nil].each do |mod|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.